src/_scss/pages/homepage/features/_featurePane.scss

```diff
@@ -0,0 +1,116 @@
+.feature-pane {
+    .feature-pane__wrapper {
+        display: block; 
+        margin: rem(60) 0 rem(100);
+        width: 100%;
+
+        @include media($medium-screen) {
+            margin: rem(120) 0 rem(60);
+        }
+    }
+
+    .feature-pane__title {
+        width: 100%; 
+        text-align: center; 
+        border: 1px solid rgba(214,215,217,0.4);
+        line-height: rem(0.1);
+        font-size: rem(16);
+        margin: 10px 0  20px;
+        color: #717171;
+
+        & span {
+            background:#fff; 
+            padding:0 10px; 
+        }
+    }
+
+    .feature-pane__content-wrapper {
+        display: flex;
```

Comment:
Our Sass code uses mixins for flexbox layout that adds vendor prefixes for older browsers. Please update to use these; for the most part, they are `@import [property]([value])`, so for example `@include display(flex)` or `@include justify-content(space-around)`